spec/acceptance/06_postgresql_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper_acceptance'
+
+describe 'sensu::backend class', unless: RSpec.configuration.sensu_cluster do
```

Comment:
How about a test that shows that after a client runs, there is expected data in the datastore. This would allow us to test from the user perspective and verify that the connection between the sensu backend and postgres are working as intended.

Reply:
I added basic check that exits 0 to generate an event and check the event with and without postgresql support enabled.